core/src/main/java/org/jdbi/v3/core/async/JdbiExecutorImpl.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.async;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.Executor;
+
+import org.jdbi.v3.core.HandleCallback;
+import org.jdbi.v3.core.HandleConsumer;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.extension.ExtensionCallback;
+import org.jdbi.v3.core.extension.ExtensionConsumer;
+import org.jdbi.v3.core.transaction.TransactionIsolationLevel;
+
+public class JdbiExecutorImpl implements JdbiExecutor {
+
+    private final Jdbi jdbi;
+    private final Executor executor;
+
+    /**
+     * Construct a {@link JdbiExecutor}.
+     *
+     * <p>
+     * The executor to pass in needs to be sized to the use case. A rule of thumb is to
+     * have the max number of workers be equal to the max number of connections in the
+     * connection pool. The worker queue in the executor should probably be bounded, unless
+     * the caller(s) already has a bound for the number of outstanding requests. Making the
+     * queue bounded will mean you are blocking the calling thread when the queue fills up.
+     * Whether or not that is a
```

Comment:
Incomplete sentence

Reply:
Ah, yes. Amended (and moved this comment block to the JdbiExecutor.create method)